### ISSUE TITLE
Unable to run on QNAP (Apache)

### ISSUE THREAD

**Issue by sw-dev1** [seblucas/cops#401]
Hi,

I am getting error 500 after install (folder cops under Web) on a QNAP Apache server.

Any special config needed for Apache?

Cheers.

**Comment by seblucas** [comment]
Did you check the logs ?

I never tried to install COPS on QNAP but I know some users did successfully.

**Comment by sw-dev1** [comment]
Yes, I checked the logs but found no error. I ended up installing the QPKG from a link here: https://forum.qnap.com/viewtopic.php?t=101794. 
This works (no 500 error). Is it a different version of your app or is it a kind of a fork for QNAP?  

Next problem is that I cannot get it to work with a Calibre database on a different server. Mapping the remote folder to a local folder via nfs gives a "database not found".

Any idea how to make a working configuration with the php app on one server and the Calibre library on another?

Merci beaucoup.
